streaming/src/main/resources/hibernate.default.properties

```diff
@@ -1,39 +0,0 @@
-# This is copied from openmrs-core GitHub repository and modified for our needs.
```

Comment:
Are you sure these are not needed anymore? I had to add them to convince the AtomFeed client to connect to DB.

Reply:
I created a template file instead for the iSantePlus work. This change should not be in this PR though